habitat-lab/habitat/config/benchmark/rearrange/hab3_bench/multi_agent_bench.yaml

```diff
@@ -0,0 +1,112 @@
+# @package _global_
```

Comment:
call this `spot_human_bench.yaml`

Reply:
both spot_spot and spot_humanoid use this as the foundation. Maybe it should stay?